Mac/Components/Browser/Album Detail/AlbumDetailSongTitleView.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  AlbumDetailSongTitleView.swift
+//  Persephone
+//
+//  Created by Nemo157 on 20200619...
+//  Copyright © 2020 Dan Barber. All rights reserved.
+//
+
+import AppKit
+
+class AlbumDetailSongTitleView: NSTableCellView {
+  @IBOutlet var songTitle: NSTextField!
+  @IBOutlet var songArtist: NSTextField!
+  var collapseArtist: NSLayoutConstraint!
+  
+  func setShowArtist(_ show: Bool) {
+    if collapseArtist == nil {
+       collapseArtist = songArtist?.heightAnchor.constraint(equalToConstant: 0.0)
```

Comment:
(Note: I'm more familiar with stack views in UIKit, so if they don't work the same way feel free to disregard!)

I wonder if putting `songTitle` and `songArtist` in a vertical stack view would simplify this code a little? It would be handy if setting `songArtist.isHidden = true` automatically collapsed the view, without having to manage constraints.

Reply:
@sharplet The vertical stack view approach works great!